### PR TITLE
Fix language in cal migration and some minor fixes

### PR DESCRIPTION
**Pull request by okmiim**
- Recreate the queryBuilder for each query, since it should not be reused ([Typo3 docs](https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Database/QueryBuilder/Index.html))
- Remove `diffsource`, since it contains an array of table fields / values which have different structure in the old and new tables
---
- Fix language reference
    - #502, #503
    - @sypets Could you please test these changes?

---
**Note**: These changes are UNTESTED!

### CODE REVIEW

**Comment by sypets** [comment]
@okmiim Great! Since we are making several changes, should we add functional tests? It is really difficult and tedious to extensively test this.

**Comment by lochmueller** [comment]
Hey @sypets & @okmiim 
I do not use the migration wizard by myself. So... feel free to test it (or write tests). On your "go" I will merge changes to the extension.... both points of @okmiim sound totally right.
Thank you,
Tim

**Review by sypets — APPROVED** [review]
@okmiim works perfectly. I migrated again and checked the results for sanity. Relations to original language look correctly (l10n_parent) and number of records with non-default translations is identical to originals in cal. Nice work and thanks!

@lochmueller Please merge. Thank you.

File: Classes/Updates/CalMigrationUpdate.php [hunk 1]
```diff
@@ -186,17 +187,31 @@ public function performCalEventUpdate($calIds, array &$dbQueries, &$customMessag
...
+                );
+            }
+
             $calendarizeEventRecord = [
                 'pid' => $event['pid'],
                 'import_id' => self::IMPORT_PREFIX . (int)$event['uid'],
                 'sys_language_uid' => $event['sys_language_uid'] ?? 0,
-                'l10n_parent' => $event['l18n_parent'] ?? 0,
-                'l10n_diffsource' => $event['l18n_diffsource'] ?? '',
+                'l10n_parent' => $parentEventUid,
```
[the remark this answers is not in the document]
Comment (sypets) [reply Classes/Updates/CalMigrationUpdate.php:214]: > I would also add 'l10n_state' => '' here.

Please ignore that. I meant to discard it. l10n_state with this PR was set to NULL, which should be ok.

**Comment by lochmueller** [comment]
Gerat. Merged!

**Pull request merged**